install_config/master_node_configuration.adoc

```diff
@@ -1832,6 +1832,36 @@ xref:../install/configuring_inventory_file.adoc#cluster-variables-table[Cluster
 
 The following examples are excerpts of redirected master log files at various log levels. System information has been removed from these examples.
 
+////
```

Comment:
@sheriff-rh I would check that this log file output is correct for the 3.11 version with the QE team (unless you have already done so.) I can't recall why this was not in there, perhaps we never had this output available for 3.11 at the time this topic was migrated to 3.11.

Reply:
Hey Vikram! I checked with @mburke5678  yesterday and we agreed to squash the output comments. My new commit this morning reflects that change. Thanks!

Reply:
@sheriff-rh - I didn't realize this was commented out section :)